### DISCUSSION TITLE
Testing connections is disabled in Airflow configuration. Contact your deployment admin to enable it.

### DISCUSSION THREAD

**Discussion by tanisha41** [apache/airflow#36553] [Q&A] [answered]
I am deploying airflow on kubernetes cluster and getting this issue 
I've set this env variable in my docker file, do i need to make changes in values.yaml ?
  ENV AIRFLOW__CORE_TEST_CONNECTION=Enabled

**Comment by potiuk** [comment, marked as answer]
You have a typo - you miss underscode between CORE and TEST

**Comment by tanisha41** [comment]
added enabled in values.yaml file 
config:
  core:
    dags_folder: '{{ include "airflow_dags" . }}'
    # This is ignored when used with the official Docker image
    load_examples: 'False'
    executor: '{{ .Values.executor }}'
    # For Airflow 1.10, backward compatibility; moved to [logging] in 2.0
    colored_console_log: 'False'
    remote_logging: '{{- ternary "True" "False" .Values.elasticsearch.enabled }}'
    test_connection: 'Enabled'               >>>> ADDED THIS
